src/steps/sites/index.test.ts

```diff
@@ -0,0 +1,79 @@
+import {
+  createMockStepExecutionContext,
+  Recording,
+} from '@jupiterone/integration-sdk-testing';
+import { IntegrationConfig } from '../../config';
+import { fetchSites } from '.';
+import { integrationConfig } from '../../../test/config';
+import { setupWPEngineRecording } from '../../../test/recording';
+import { fetchAccounts } from '../accounts';
+
+describe('#fetchSite', () => {
+  let recording: Recording;
+
+  beforeEach(() => {
+    recording = setupWPEngineRecording({
+      directory: __dirname,
+      name: 'fetchSite',
+    });
+  });
+
+  afterEach(async () => {
+    await recording.stop();
+  });
+
+  test('should collect data', async () => {
+    const context = createMockStepExecutionContext<IntegrationConfig>({
+      instanceConfig: integrationConfig,
+    });
+
+    await fetchAccounts(context);
+    await fetchSites(context);
```

Comment:
You should call this (in describe) the same as the main step function you're calling (we're testing it) - so change `#fetchSite` to `#fetchSites` (and same for L17 for recording name directory).